srcdoc/configure/syntax_and_structure.mt

```diff
@@ -125,4 +125,39 @@ hosts:
 EOT
 ?>
 
+<h3 id="include_other_files">Using Inclusion</h3>
+
+<p>
+Since version 2.1, H2O makes it possible to include other config files using <code>!file</code> custom YAML tag.
```

Comment:
s/H2O makes it possible to include other config files/it is possible to include a YAML file from the configuration file/ ?